docs/en/di-container-pgsql.md

```diff
@@ -0,0 +1,49 @@
+## Create connection with di container for pgsql server
```

Comment:
```suggestion
# Connecting PostgreSQL
```

I'd generalize such connection recipes and include both DI container instructions and no-container instructions into it.